components/Article/index.js

```diff
@@ -34,20 +49,6 @@ function Article({ article }) {
 
 export default Article;
 
-const StyledArticle = styled.article`
-  border: 1px solid black;
-  margin: 1rem 1rem 50px 1rem;
-  list-style: none;
-  position: relative;
-  background-color: white;
-  border: none;
-  border-radius: 2px;
-`;
-
-const StyledDetailsLink = styled(Link)`
-  position: absolute;
-  right: 10px;
-  top: 10px;
-  border-style: none;
-  color: inherit;
+const StyledSpan = styled.span`
+  tab-size: 100px;
 `;
```

Comment:
Never seen, never heard before - what does this property do and why do you need this? 😬

Reply:
it was an attempt to indent the text. Forgotten to delete... I could solve it with grid.